src/components/DropDownbox.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="dropdown">
+    <label for="optionName">{{ optionName }}: &nbsp; </label>
+    <select id="optionName" name="optionName" @change="handleChange">
+      <option value="">--Please choose an option--</option>
+      <option
+        v-for="option in options"
+        :key="option.value"
+        :value="option.value"
+      >
```

Comment:
I think it's more useful if there is a default value already in. Like for example, if the user wants to keep taking the quiz over and over again, it can be pretty annoying trying to alter the quiz time.
Something like
```tsx
<select :value="options[0] ?? ''" id="optionName" name="optionName" @change="handleChange">
```
Note the `?? ''`. This is important in the event that if there are no elements in the options, which **should be impossible**, it will still provide a default value. This is just in case if we need to reuse this component for other functionality